resolwe/flow/managers/listener/__init__.py

```diff
@@ -0,0 +1,12 @@
+""".. Ignore pydocstyle D400.
+
+========
+Listener
+========
+
+"""
+from .listener import ExecutorListener
+
+__all__ = ("listener", ExecutorListener)
+
+listener = ExecutorListener()
```

Comment:
Why do you initialize `listener` here and also in `resolwe/flow/managers/__init__.py`?

Reply:
I have removed the initialization from this `__init__.py` file.